stable/spring-cloud-data-flow/README.md

```diff
@@ -57,6 +57,12 @@ Only one messaging layer can be used at a given time. If RabbitMQ and Kafka are
 
 Note that this chart pulls in many different Docker images so can take a while to fully install.
 
+### Disabling Skipper Server
```

Comment:
Similar to how someone can negate streaming features entirely, it'd be helpful also to give the choice to disable task/batch features, as well. 

Perhaps we could have two flags. 1) `streaming.enabled` and 2) `batch.enabled`.

When `streaming.enabled` and `rabbitmq.enabled` are set to "false," then everything in this PR would take into account.

Likewise, when `batch.enabled` is set to "false", we could ignore the task's platform account configuration (see [here](https://github.com/helm/charts/blob/master/stable/spring-cloud-data-flow/templates/server-config.yaml#L14-L24)), and as well as negate "schedules" feature from being enabled (see [here](https://github.com/helm/charts/blob/master/stable/spring-cloud-data-flow/templates/server-deployment.yaml#L63-L64)).

Lastly, no one should be allowed to disable both features. At least one of them should be required to use the chart. :)

Reply:
should be all set - with the exception of the disable check, not really a clean way to do that..